package.json

```diff
@@ -36,8 +36,10 @@
     "@types/react-dom": "^16.0.9",
     "@types/react-helmet-async": "^0.0.1",
     "@types/react-hot-loader": "^4.1.0",
+    "@types/react-lottie": "^1.2.0",
```

Comment:
do we need this now that we have our own lottie types?

Reply:
Good catch!